src/foam/u2/wizard/debug/TestWizard.js

```diff
@@ -20,22 +20,31 @@ foam.CLASS({
   ],
 
   properties: [
+    {
+      class: 'StringArray',
+      name: 'packages',
+      factory: () => ['foam.u2.wizard.debug.scenarios']
+    },
     {
       name: 'children_',
       factory: function () {
-        // ???: is it possible to ask foam for all models extending TestWizardScenario?
-        const scenarios = Object.getOwnPropertyNames(foam.u2.wizard.debug.scenarios);
-        
-        return this.ArrayDAO.create({
-          array: scenarios.map(scenarioName => {
-            return this.Menu.create({
+        const scenarioMenus = [];
+
+        for ( const packageString of this.packages ) {
+          const pkg = packageString.split('.').reduce((o, k) => o[k], globalThis);
+          scenarioMenus.push(...Object.getOwnPropertyNames(pkg).map(scenarioName =>
```

Comment:
Nice chaining! should we check and log if the chaining is unsuccessful?

Reply:
Yeah I was really proud of this line :joy: that's a good idea, working on it now